docs/getting-started/DebuggingGuide.md

```diff
@@ -0,0 +1,55 @@
+# From Zero to Bug Fix Hero
```

Comment:
@VCOLAWTON, this is the accumulation of my notes from the WoW session last week. I'd appreciate any feedback.

Reply:
@marmegh @vcolawton @eddynaka 

Please close on this when you can. Thanks!